animareflection/ui
# Show + Hide Components

## Description

##### Task links:  [Show](https://trello.com/c/FlMkOS9q/118-show-component) and [Hide](https://trello.com/c/hjHAN2oK/109-hide-component)

Added `Show` and `Hide` components to conveniently set `display` to `none` for certain elements. These components are responsive around defined breakpoints (i.e. `md`, which the default for is `768px`).

## Test Steps

1) Verify component, recipe, and story structure is sound.
2) Verify that components render and are responsive around breakpoints correctly in both storybook and example apps.
